GermanFilter/sections/css_extended.txt

```diff
@@ -6,6 +6,12 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/139373
+rp-online.de#?#section > article div[data-article-content=""] ~ div:has(.lazyad)
```

Comment:
Possible to make ext-css bit more detailed in path.
Applies to all ext-CSS rules.

Reply:
How much you want rules more detailed?

Reply:
I suggest to use these rules (my top rules are a bit safer than your rp-online.de##main aside[class*="print:hidden"][data-cy="portal-top"]):

```
rp-online.de#?#aside[data-cy="portal"]:has(> div[id^="ir_rpd_"])
rp-online.de#?#div[class]:has(> aside[data-cy="portal"][style="display: none !important;"]:only-child)
rp-online.de#?#.w-full:has(> .flex:only-child > .park-content-list-body[style="display: none !important;"]:only-child)
rp-online.de#?#main > section[data-cy="module"].w-full:has(> header > div[class] > .bg-special-ad)
rp-online.de#?#main > section[data-cy="section"].w-full:has(> div[data-cy="section-weekli-header"])
! article top
rp-online.de##div[id^="ir_rpd"]
rp-online.de#?#aside:has(> div[id^="ir_rpd"])
```

Also, glomex player has sometimes anti-adblock warning, element hiding rule exclusion fixes the issue (German filter -> antiblock -> ! glomex player adblock detection)

Reply:
OK.